cli/src/cmd_auth.rs

```diff
@@ -535,6 +544,23 @@ impl CmdAuthStatus {
     }
 }
 
+/// Create a file. On Unix set permissions to 0600.
+fn create_private_file(path: &Path) -> Result<File> {
+    let file = File::create(path)?;
```

Comment:
I think this leaves a race where one could open the file before the permissions were changed.

Reply:
Good callout, I've switched to `OpenOptions` to set the mode on creation.